work/Maya/exercise3_1.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+import threading
+from time import time
+
+data = {'A': [1, 2, 3, 4],
+        'B': [5, 6, 7, 8],
```

Comment:
the code I provide have a dataset more big, but okey

Reply:
I couldn't find the dataset you provided. Is it on teams or github?